### ISSUE TITLE
Wine does not work

### ISSUE THREAD

**Issue by derryktutt** [Botspot/pi-apps#1419]
### Confirmations

- [X] I searched for similar bug reports in https://github.com/Botspot/pi-apps/issues and found none were relevant.
- [X] I tried updating Pi Apps and the issue is still not fixed. [How to update Pi Apps?](https://github.com/Botspot/pi-apps#to-update-pi-apps)

### What happened?

Wine does not run anything

### Description

When I run `wine` it responds with `setarch: failed to execute /opt/wine/bin/wine: no such file or directory` 

### What is your operating system?

Raspberry Pi OS 11, armhf

**Comment by Botspot** [comment]
Could you go to the Pi-Apps Log Viewer, find the logs from when you installed wine, and send that to me?
Perhaps a download failed but the script did not exit with an error.

**Comment by Rak1ta** [comment]
@Botspot hello! I recently created a discussion here about the work of wine. Hope you noticed it. [click](https://github.com/Botspot/pi-apps/issues/1391). 
---
Since the installation files in Pi-Apps weigh kilobytes, is it possible to enter this into the program? - Upon successful installation, the installation script is saved to a folder .. for example "apps_passed". If an update has occurred and it has an error, the program suggests using the previous installation script.
Or there was a bug, for example Angry IP scanner was not installed due to "without zst compression support". Using the button (see screenshot), I would restore the previous version.

I did not want to create an additional discussion, and there are so many open ones hanging. This is a suggestion for a similar situation. Thanks!
![Безымянный](https://user-images.githubusercontent.com/88277343/150649719-674e2d69-f3cd-48a0-9703-f02c68c1d0a3.png)
